### ISSUE TITLE
How do you think ?

### ISSUE THREAD

**Issue by ZiaRek89** [diegopamio/angular-sails-bind#13]
Hey, how do you think, the same lib can be done with Laravel? I mean Laravel is very similiar to Sails.

**Comment by apinnecke** [comment]
I don't think so. Laravel is a PH Framework without socket Support, it's based on pure request based PHP.
